client/state/push-notifications/reducer.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import debugFactory from 'debug';
+import moment from 'moment';
+import omit from 'lodash/omit';
+import pick from 'lodash/pick';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DESERIALIZE,
+	PUSH_NOTIFICATIONS_API_READY,
+	PUSH_NOTIFICATIONS_AUTHORIZE,
+	PUSH_NOTIFICATIONS_BLOCK,
+	PUSH_NOTIFICATIONS_DISMISS_NOTICE,
+	PUSH_NOTIFICATIONS_MUST_PROMPT,
+	PUSH_NOTIFICATIONS_RECEIVE_REGISTER_DEVICE,
+	PUSH_NOTIFICATIONS_RECEIVE_UNREGISTER_DEVICE,
+	PUSH_NOTIFICATIONS_TOGGLE_ENABLED,
+	PUSH_NOTIFICATIONS_TOGGLE_UNBLOCK_INSTRUCTIONS,
+	SERIALIZE,
+} from 'state/action-types';
+const debug = debugFactory( 'calypso:push-notifications' );
+
+const UNPERSISTED_SYSTEM_NODES = [
+	'apiReady',
+	'authorized',
+	'authorizationLoaded',
+	'blocked',
+];
+function system( state = {}, action ) {
+	switch ( action.type ) {
+		// System state is not persisted
+		case DESERIALIZE: {
+			const newState = omit( state, UNPERSISTED_SYSTEM_NODES );
+			return omit( newState, UNPERSISTED_SYSTEM_NODES );
+		}
+
+		case SERIALIZE: {
+			const newState = omit( state, UNPERSISTED_SYSTEM_NODES );
```

Comment:
`newState` is redundant here, and there are 2 similar cases further down in this file.